src/components/NavBar.module.css

```diff
@@ -0,0 +1,55 @@
+.root {
+  --icon-font: montserrat;
+  --font: robotoslab;
+}
+
+.NavBar {
+  display: flex;
+  justify-content: flex-start;
+  padding: 2px 5px;
+  align-items: center;
+  height: 3rem;
+  margin-bottom: 1rem;
+  background-color: #fff;
+  margin-top: 1rem;
+}
+
+.navLinks {
+  display: flex;
+  list-style-type: none;
+  text-decoration: none;
+}
+
+.BookstoreTitle {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: var(--icon-font);
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.BOOKS {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: var(--icon-font);
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-transform: uppercase;
+  text-decoration: none;
+}
```

Comment:
- [ ] DRY rule right here 👍 
_Please see the comment above about the DRY rule._